addons/test_mass_mailing/tests/test_blacklist_behavior.py

```diff
@@ -91,3 +91,41 @@ def _test_mailing_bounce_w_auto_bl(self, bounce_base_values):
             [{'email': 'test.record.00@test.example.com', 'trace_status': 'cancel', 'failure_type': 'mail_bl'}],
             new_mailing, target, check_mail=True
         )
+
+
+@tagged('mail_blacklist')
+class TestBlackListAlwaysApplied(common.TestMassMailCommon):
+    """ Test that blacklist is applied even if the target model doesn't inherit from mail.thread.blacklist.
+
+    Note: special case not tested to not introduce model change as a fix: target model with partner_ids field
```

Comment:
I don't think this comment is relevant here (because it's already mentioned in the commit message)

Reply:
Yes indeed, especially as it is a patch that will reworked in master; I have removed the node. Thanks